tests/PactNet.Tests/Drivers/FfiIntegrationTests.cs

```diff
@@ -24,28 +27,29 @@ public FfiIntegrationTests(ITestOutputHelper output)
             this.output = output;
 
             NativeInterop.LogToBuffer(LevelFilter.Trace);
-        }
-
+        }
+
         [Fact]
-        public async Task HttpInteraction_v3_CreatesPactFile()
```

Comment:
The line-endings problem has really confused the diff here. I can't really follow this as-is because it thinks an existing large integration test was rewritten instead of a new test added.

Reply:
Sorry Adam, I think I have fixed the issue now